src/test/regress/expected/prepared_statements_1.out

```diff
@@ -0,0 +1,143 @@
+SET search_path TO "prepared statements";
+PREPARE prepared_test_1 AS
```

Comment:
repartition jobs are creating too much load when we have too many nodes / shards. It even times out.

I'll see what we can do about that

Reply:
Interesting, even when that doesn't time out, it might make the tests take a very long time to complete. Have you found any ideas about this yet? If not, I will think as well